### ISSUE TITLE
Cannot import Gwaihir: cannot allocate memory in static TLS block

### ISSUE THREAD

**Issue by DSimonne** [DSimonne/gwaihir#55]
After typing:
```python
from gwaihir.gui import gui
```
One gets the following error:

```python
ImportError: /lib/powerpc64le-linux-gnu/libgomp.so.1: cannot allocate memory in static TLS block
```

We tested different step by step imports:
* Importing directly Gwaihir does not work

But this works:
* Restart the kernel
* Import xrayutilites
* import vtk
* import gwaihir

Why ?

**Comment by DSimonne** [comment]
Only seem to happen on the p9, needs to test at SixS to see if it is really the case

**Comment by ccechatelier** [comment]
Good point indeed. From what I read on the Internet, it could be also linked to some openmp configuration issues. We might need to open a trouble ticket at ESRF.

**Comment by DSimonne** [comment]
Yes, Vincent is on it, he said it could be nice if we manage to reproduce the error with a few imports.

I think it is the same in the terminal as well , but always linked to the import of Gwaihir

**Comment by ccechatelier** [comment]
Yes it's the same error while loading in a terminal. Importing xrayutilities only does the same error as well !
